### ISSUE TITLE
No h1 tag on homepage

### ISSUE THREAD

**Issue by EvanHerman** [godaddy-wordpress/go#325]
**Describe the bug:**
There is no `h1` tag on the homepage meaning we do not meet WCAG 2.0 requirements, and is causing our `a11y` tests to fail. If we want to use the `accessibility` tag when we submit the theme, we need to include an `h1` tag on the homepage. 

**Expected behavior:**
The homepage should contain an `h1` tag.

**Priority** (select one):
- [x] High - Functional blocker
- [ ] Medium - Doesn't work as expected 
- [ ] Low - Not a big deal but we'd want to get to this eventually
